.rubocop_todo.yml

```diff
@@ -1,55 +1,37 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2024-02-09 08:47:26 UTC using RuboCop version 1.50.2.
+# on 2024-06-09 12:47:09 UTC using RuboCop version 1.63.5.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
 # Offense count: 1
 # This cop supports safe autocorrection (--autocorrect).
-# Configuration parameters: Severity, Include.
-# Include: **/*.gemspec
-Gemspec/RequireMFA:
+# Configuration parameters: Width, AllowedPatterns.
+Layout/IndentationWidth:
```

Comment:
I can work on cleaning this up after the PR got accepted. I didn't want to invest too much time right now.